### PR TITLE
[#2223] Fix documentation according to set guidelines.

### PR DESCRIPTION
**Pull request by csurfer**
This PR
1. Removes the remaining `.. code-block:: pycon` references in docs.
2. Removes the remaining `>>>` references in docs.
3. Changes the documentation in accordance with set `import` guidelines.
4. Adds `print` comments where appropriate

Closes https://github.com/urllib3/urllib3/issues/2223

### CODE REVIEW

**Comment by csurfer** [comment]
PTAL @sethmlarson

File: docs/user-guide.rst [hunk 1]
```diff
@@ -228,11 +228,11 @@ in the URL:
 .. code-block:: python
 
     import json
+    import urllib
     import urllib3
-    from urllib.parse import urlencode
 
     # Encode the args into url grammer.
```
Comment (sethmlarson) [inline docs/user-guide.rst:234]: Can we fix this typo "grammar"

File: docs/index.rst [hunk 2]
```diff
@@ -31,14 +31,14 @@ standard libraries:
...
 
-.. code-block:: pycon
-
-   >>> import urllib3
-   >>> resp = urllib3.request("GET", "https://httpbin.org/robots.txt")
-   >>> resp.status
-   200
-   >>> resp.data
-   "User-agent: *\nDisallow: /deny\n"
+.. code-block:: python
```
Comment (sethmlarson) [inline docs/index.rst:34]: Similarly to `README` we'll keep this one.

File: docs/advanced-usage.rst [hunk 3]
```diff
@@ -227,9 +227,9 @@ Once PySocks is installed, you can use
 
 .. code-block:: python
 
-    from urllib3.contrib.socks import SOCKSProxyManager
+    import urllib3
 
-    proxy = SOCKSProxyManager("socks5h://localhost:8889/")
+    proxy = urllib3.contrib.socks.SOCKSProxyManager("socks5h://localhost:8889/")
```
Comment (sethmlarson) [inline docs/advanced-usage.rst:232]: So this example can stay the same because you can't use `urllib3.contrib.*` directly. Let's leave this as-is.

**Review by sethmlarson — CHANGES_REQUESTED** [review]
Few comments, unfortunately reduces the PR quite a lot but I think this means we can close the parent issue?

File: README.rst [hunk 4]
```diff
@@ -30,14 +30,14 @@ standard libraries:
...
 
-.. code-block:: pycon
-
-   >>> import urllib3
-   >>> resp = urllib3.request("GET", "https://httpbin.org/robots.txt")
-   >>> resp.status
-   200
-   >>> resp.data
-   "User-agent: *\nDisallow: /deny\n"
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (csurfer) [reply README.rst:33]: Converted to `pycon` code block but removed the `>>>` and retained the print comments.

File: docs/user-guide.rst (hunk 1 above)
Comment (csurfer) [reply docs/user-guide.rst:234]: Done.

File: docs/index.rst (hunk 2 above)
Comment (csurfer) [reply docs/index.rst:34]: Similar to `README` brought back the `pycon` block but removed the `>>>` and retained the print comments.

File: docs/advanced-usage.rst (hunk 3 above)
Comment (csurfer) [reply docs/advanced-usage.rst:232]: Done.

**Comment by csurfer** [comment]
@sethmlarson Anything else that needs to be done here?

**Review by sethmlarson — APPROVED** [review]
Thanks! I modified things to look how we discussed in Discord. We want to keep the `>>>` in the index examples.

**Pull request merged**